src/core/compact_object.cc

```diff
@@ -1265,7 +1258,7 @@ void CompactObj::Materialize(std::string_view blob, bool is_raw) {
       u_.r_obj.SetString(blob, tl.local_mr);
     }
   } else {
-    EncodeString(blob, false);
+    SetString(blob, false);
```

Comment:
Actually the size after editing an offlaoded string can be smaller, so we have to call SetString

Reply:
No... there is no command that makes it shrink 🤦🏻